pkgs/os-specific/linux/criu/default.nix

```diff
@@ -39,6 +40,7 @@ stdenv.mkDerivation rec {
   postFixup = ''
     wrapProgram $out/bin/criu \
       --prefix PATH : ${lib.makeBinPath [ iptables ]}
```

Comment:
please use `makeWrapperArgs` to avoid double-wrapping

Reply:
In this case it should be fine since `criu` is an ELF binary:
```
├── bin
│   ├── compel
│   ├── crit
│   ├── .crit-wrapped
│   ├── criu
│   └── .criu-wrapped
```

Or am I missing anything?

Reply:
hmm, odd, I thought it would wrap twice now.